src/uniprotkb/components/entry/Entry.tsx

```diff
@@ -322,6 +333,9 @@ const Entry: FC = () => {
           }
           id={TabLocation.Publications}
         >
+          <Helmet>
+            <title>{pageTitle} | Publications</title>
```

Comment:
Not sure here, I'd say the most important thing would be to have the entry name first as it's more likely to be help discrimate between 2 pages in the history or in tabs rather than seeing `Publicati...` and not being able to tell which one is which. Within one entry, it's easy enough to switch to the correct tab if that wasn't the right page that you opened, whereas if it's not the same entry it's more complicated.
I did have doubts about switching namespace and subPage though, I don't know what you think about that.

Reply:
I see what you mean about the entry being most important. How about:
```
APP - Amyloid-beta precursor protein - Homo sapiens (Human) | Publications | UniProtKB
```